mod_form.php

```diff
@@ -163,7 +163,7 @@ public function definition_after_data() {
      * @return bool
      */
     public function completion_rule_enabled($data): bool {
-        return (!empty($data['completiontype']) && $data['completiontype'] > 0);
+        return (!empty($data['completiontype' . $this->get_suffix()]));
```

Comment:
is the check for the completiontype > 0 not needed anymore? And if so, is this also downwards compatible to moodle 4.2 and before?

Reply:
This is an obsolete check which can be removed.